<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registratew</class>
 <widget class="QWidget" name="registratew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #bdbdbd;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QPushButton" name="avatar">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #111111;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_change_by_url">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #cdcdcd;
	border: 2px solid #F9F9F9;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #111111;
	border-right-color: #111111;
	color: #111111;
 }
QPushButton:pressed {
	background: #aaaaaa;
	border: 2px solid #111111;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #333333;
 }
QPushButton:hover {
	background: #aaaaaa;
	color: #333333;
 }</string>
       </property>
       <property name="text">
        <string>Смена по URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_change_by_file">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #cdcdcd;
	border: 2px solid #F9F9F9;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #111111;
	border-right-color: #111111;
	color: #111111;
 }
QPushButton:pressed {
	background: #aaaaaa;
	border: 2px solid #111111;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #333333;
 }
QPushButton:hover {
	background: #aaaaaa;
	color: #333333;
 }</string>
       </property>
       <property name="text">
        <string>Смена файлом</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_id">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_username">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Имя пользователя</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_currency">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Валюта</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_balance">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Баланс</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_email">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>e-mail</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_sex">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Пол</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_phone">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Телефон</string>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_regdate">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string>Дата регистрации</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_id_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_username_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_currency_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_balance_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_email_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_sex_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_phone_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_regdate_value">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #111111;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
